### PR TITLE
Fix srpm API performance that resulted in timeouts

### PR DESCRIPTION
**Pull request by matysek**
This fixes the timeout in /names/srpms/<name> and optimizes response time. On my local setup I got response time for kernel package name from 102 sec to 8 sec.

### CODE REVIEW

File: webapp/srpm_pkg_names.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ class SRPMPkgNamesAPI:
 
     def __init__(self, cache):
         self.cache = cache
-
+    @profile
```
Comment (tkasparek) [inline webapp/srpm_pkg_names.py:15]: This line shouldn't be in production code.

Comment (matysek) [reply webapp/srpm_pkg_names.py]: Yes, I forgot it.

**Comment by matysek** [comment]
@tkasparek and the post api does work for you? I tested it using the post api in /api/ui

**Comment by matysek** [comment]
@tkasparek I tested it myself locally and it DOES work for me. There has to be something with your dev environment.
```
time curl -X GET "http://localhost:8080/api/v1/package_names/srpms/kernel" -H  "accept: application/json"
{"last_change": "2020-05-26T19:08:21.829732+00:00", "srpm_name_list": {"kernel": {"rhel-5-for-power-beta-rpms": ["kernel", "kernel-debug", "kernel-debug-devel", "kernel-devel", "kernel-doc", "kernel-headers", "kernel-kdump", "kernel-kdump-devel"], "rhel-7-for-power-beta-rpms": ["bpftool", "kernel", "kernel-abi-whitelists",
...
"rhel-7-server-aus-rpms": ["bpftool", "kernel", "kernel-abi-whitelists", "kernel-debug", "kernel-debug-devel", "kernel-devel", "kernel-doc", "kernel-headers", "kernel-tools", "kernel-tools-libs", "kernel-tools-libs-devel", "perf", "python-perf"], "rhel-5-server-els-rpms": ["kernel", "kernel-PAE", "kernel-PAE-devel", "kernel-debug", "kernel-debug-devel", "kernel-devel", "kernel-doc", "kernel-headers", "kernel-xen", "kernel-xen-devel"], "rhel-7-for-arm-64-rhui-optional-rpms": ["kernel", "kernel-abi-whitelists", "kernel-doc", "kernel-tools-libs-devel"]}}}curl -X GET "http://localhost:8080/api/v1/package_names/srpms/kernel" -H   0.01s user 0.01s system 0% cpu 5.890 total
```

**Pull request merged**